packages/shared/styles/components/atoms/SfDropdown.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../helpers";
+
+.sf-dropdown {
+  position: var(--dropdown-position, fixed);
+  width: var(--dropdown-width, 100%);
+  top: var(--dropdown-top);
+  bottom: var(--dropdown-bottom, 0);
+  transform: var(--dropdown-transform);
+  @include for-desktop {
+    position: var(--dropdown-position, absolute);
+    transform: var(--dropdown-transform, translate(0, 100%))
+  }
+  &__container {
+    position: relative;
+    z-index: 1;
+    background: var(--dropdown-background, var(--c-white));
+    box-shadow: var(--dropdown-box-shadow);
+    @include for-desktop {
```

Comment:
how about writing desktop styles in one block at the end of sf-dropdown block ? wdyt ?

Reply:
IMO we should use element just one time. Write styles for it on desktop inside it. Why? Styles for this element is near and you can see what you should override or what you shouldn't override